proto/dydxprotocol/clob/query.proto

```diff
@@ -183,6 +187,7 @@ message StreamUpdate {
     StreamOrderbookUpdate orderbook_update = 1;
     StreamOrderbookFill order_fill = 2;
     StreamTakerOrder taker_order = 3;
+    StreamSubaccountUpdate subaccount_update = 6;
```

Comment:
i think we can just put this as 4 and shift everything else.

Reply:
b/c doesnt need to be backwards compatible

Reply:
done